libraries/src/Console/DeleteUserCommand.php

```diff
@@ -86,7 +86,7 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
 			return 1;
 		}
 
-		if ((!$this->ioStyle->confirm('Are you sure you want to delete this user?', false)) && $input->isInteractive())
+		if ($input->isInteractive() && (!$this->ioStyle->confirm('Are you sure you want to delete this user?', false)))
```

Comment:
```suggestion
		if ($input->isInteractive() && !$this->ioStyle->confirm('Are you sure you want to delete this user?', false))
```
These brackets are not really necessary due to precedence of operators, and also not help with readability.